astropy/units/core.py

```diff
@@ -756,8 +756,12 @@ def to_string(self, format=unit_format.Generic, **kwargs):
 
         **kwargs :
             Further options forwarded to the formatter. Currently
-            recognized is **inline** (:class:`bool`) for the
-            ``"latex"``, ``"console"``, and``"unicode"`` formats.
+            recognized is **fraction** (:class:`bool`), which determines
+            whether a unit with bases raised to negative powers is
+            represented using a fraction or not, and **inline**, which
+            determines whether the fraction is set with a solidus, or
+            displayed with a numerator and denominator (the latter only
```

Comment:
Re your comment about the behavior of `inline`, I do think it should error, not ignore.

Reply:
OK, happy to implement that. What do you think about the `fraction={'inline'|'display'|True|False}` alternative? In the current wave of my wavering, I like that a little more, as it keeps things together while for `fraction=False`, `inline` has no meaning.

Reply:
I like the idea of consolidating. In particular, this format is more easily extended in the future, adding more Literal options beyond 'inline' and 'display'.

Reply:
Fraction options are now consolidated and an exception is raised if unknown, so I think the above was addressed. (Not the move to tuples, since in this PR, the formats still sort.)